src/firewall/firewalld.rs

```diff
@@ -67,21 +66,24 @@ impl firewall::FirewallDriver for FirewallD {
 
     fn setup_port_forward(
         &self,
+        _network: Network,
         _container_id: &str,
         _port_mappings: Vec<types::PortMapping>,
-        _container_ip: IpAddr,
-        _network: IpNet,
         _network_name: &str,
         _id_network_hash: &str,
+        _options: &PerNetworkOptions,
     ) -> Result<(), Box<dyn Error>> {
         todo!();
     }
 
     fn teardown_port_forward(
         &self,
+        _network: Network,
         _container_id: &str,
         _port_mappings: Vec<types::PortMapping>,
-        _container_ip: &str,
+        _network_name: &str,
+        _id_network_hash: &str,
+        _options: &PerNetworkOptions,
```

Comment:
Please remove the leading underscore from the variables so that the compiler will complain if they are unused.